sdk/burdock/reader/sql/databricks.py

```diff
@@ -0,0 +1,97 @@
+import datetime

```

Comment:
Awesome PR.  Just to get an idea, is this databricks specific because of some nuances in the implementation or could this work on other clusters? I am trying to see if this should be named SparkReader or not.

Reply:
Thanks @eedeleon you're right, it makes sense. 
I named it as databrickSparkReader just because SparkReader already added by @joshua-oss :), it is the old one we developed previously and it only works with a dataset size fits in memory, not as good as the new one we add now. But it matches everything we have in new semantics right now, 
and I can't pass dummy None to PrivateReader, I have to use dummy SparkReader, I made a new commit to fix it, basically use SparkReader in databrickSparkReader. 
Probably we will have both in parallel for now.

Do you have other suggestion on the name?